.ci/release.sh

```diff
@@ -38,22 +38,30 @@ sed "s~image: jaegertracing/jaeger-operator.*~image: ${BUILD_IMAGE}~gi" -i test/
 # change the versions.txt
 sed "s~${PREVIOUS_VERSION}~${OPERATOR_VERSION}~gi" -i versions.txt
 
+mkdir -p deploy/olm-catalog/jaeger-operator/${OPERATOR_VERSION}
+cp deploy/olm-catalog/jaeger-operator/manifests/jaeger-operator.clusterserviceversion.yaml \
+   deploy/olm-catalog/jaeger-operator/${OPERATOR_VERSION}/jaeger-operator.v${OPERATOR_VERSION}.clusterserviceversion.yaml
+
 operator-sdk generate csv \
     --csv-channel=stable \
     --make-manifests=false \
-    --csv-version=${OPERATOR_VERSION} \
-    --from-version=${PREVIOUS_VERSION}
```

Comment:
Make sure you run it twice: one for the version you want right now, and one for the next version, to see if the current changes will work in the future. Just don't forget to revert those changes before committing :-)

Reply:
When I specified `--from-version,` the generation command tries to find a manifest to update at `deploy/olm-catalog/jaeger-operator/{PREVIOUS_VERSION}/jaeger-operator.{PREVIOUS_VERSION}.clusterserviceversion.yaml.`. That approach could work if we make sure that we are going to have all the previous manifests. In this case because we don't have the manifest for 0.19.0 the generation command asks for all operator metadata, (displayName, description etc...) in interactive mode, that situation makes our release script fails with an EOF error:
```

Releasing 1.20.0 from release/v1.20.0
time="2020-09-30T10:51:32Z" level=info msg="Generating CSV manifest version 1.20.0"
2020/09/30 10:51:32 Error when reading input: EOF

```

Reply:
That will fail for 1.20.0, but what about 1.21.0 ?

Reply:
It should not fail if we have the previous version, and if we can make sure that it will be always the case.

Reply:
Please, go over the exercise of creating the next version as well. There might be other issues that we are not accounting for.

Reply:
I tested this  trying to generate the csv for 1.21.0, it works.

 Also I realized that we don't need to commit the `deploy/olm-catalog/jaeger-operator/{OPERATOR_VERSION}/jaeger-operator.{OPERATOR_VERSION}.clusterserviceversion.yaml` file, as we are going to have the latest YAML on `deploy/olm-catalog/jaeger-operator/manifests/jaeger-operator.clusterserviceversion.yaml` we are going to use that file for the next release. But not sure if we would like to have the old manifests, . WDYT @jpkrohling